lib/core/src/irods_parse_command_line_options.cpp

```diff
@@ -277,7 +277,7 @@ static int build_irods_path_structure(const path_list_t& _path_list,
         return USER__NULL_INPUT_ERR;
     }
 
-    memset( _rods_paths, 0, sizeof( rodsPathInp_t ) );
+    clearRodsPathInp(_rods_paths);
```

Comment:
Does the clear operation reset/memset the input for reuse?

Reply:
`clearRodsPathInp(...)` does memset the input. I based it off of the other `clear...` functions.

Reply:
Got it.